modules/strelka/1.0/strelka.smk

```diff
@@ -0,0 +1,213 @@
+#!/usr/bin/env snakemake
```

Comment:
The Strelka user guide has the same recommendation as Manta for excluding the small contigs. Could you try implementing that just like what I did with Manta. 

https://github.com/Illumina/strelka/blob/v2.9.x/docs/userGuide/README.md#improving-runtime-for-references-with-many-short-contigs-such-as-grch38